docs/package.json

```diff
@@ -19,8 +19,8 @@
     "@mdx-js/react": "^1.6.22",
     "clsx": "^1.1.1",
     "prism-react-renderer": "^1.3.3",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "react": "^18.2.0",
```

Comment:
Can you revert this change? Our CI is skipping because you've edited the docs (not ideal)

Reply:
I removed it from the commit.

Reply:
But I introduced another problem - I'm on it.